isis/tests/FunctionalTestsCaminfo.cpp

```diff
@@ -16,155 +16,298 @@ static QString APP_XML = FileName("$ISISROOT/bin/xml/caminfo.xml").expanded();
+        QString stringCsvValue = currentColumn[0];
+        bool isCsvNumeric = false;
+        double numericCsvValue = stringCsvValue.toDouble(&isCsvNumeric);
+        QString stringPvlValue = QString(currentKey);
+        bool isPvlNumeric = false;
+        double numericPvlValue = stringPvlValue.toDouble(&isPvlNumeric);
+        if (isCsvNumeric && isPvlNumeric) {
+            EXPECT_NEAR(numericCsvValue, numericPvlValue, 0.001) <<
+                    "Column [" << columnName.toStdString() << "] value [" <<
+                    stringCsvValue.toStdString() << "] does not match Pvl value [" <<
+                    stringPvlValue.toStdString() << "] within tolerance [0.001].";
+        }
+        else {
+            EXPECT_EQ(stringCsvValue.toStdString(), stringPvlValue.toStdString()) <<
+                    "Column [" << columnName.toStdString() << "] value [" <<
+                    stringCsvValue.toStdString() << "] does not match Pvl value [" <<
+                    stringPvlValue.toStdString() << "].";
+        }
+    }
+
+    for (int i = 0; i < geometry.keywords(); i++) {
+        PvlKeyword currentKey = geometry[i];
+        QString columnName = "Geom_" + currentKey.name();
+        CSVReader::CSVAxis currentColumn = caminfoCsv.getColumn(columnName);
+        EXPECT_TRUE(currentColumn.dim1() > 0) << "Failed to find column [" <<
+                                                 columnName.toStdString() << "] in CSV";
+        QString stringCsvValue = currentColumn[0];
+        bool isCsvNumeric = false;
+        double numericCsvValue = stringCsvValue.toDouble(&isCsvNumeric);
+        QString stringPvlValue = QString(currentKey);
+        bool isPvlNumeric = false;
+        double numericPvlValue = stringPvlValue.toDouble(&isPvlNumeric);
+        if (isCsvNumeric && isPvlNumeric) {
+            EXPECT_NEAR(numericCsvValue, numericPvlValue, 0.001) <<
+                    "Column [" << columnName.toStdString() << "] value [" <<
+                    stringCsvValue.toStdString() << "] does not match Pvl value [" <<
+                    stringPvlValue.toStdString() << "] within tolerance [0.001].";
+        }
+        else {
+            EXPECT_EQ(stringCsvValue.toStdString(), stringPvlValue.toStdString()) <<
+                    "Column [" << columnName.toStdString() << "] value [" <<
+                    stringCsvValue.toStdString() << "] does not match Pvl value [" <<
+                    stringPvlValue.toStdString() << "].";
+        }
+    }
+
+    // CSVReader::CSVAxis csvLine;
+    // CSVReader::CSVAxis csvColumn;
+    // // Access the "Header" information first
+
+    // csvColumn = header.getColumn();
+
+    // // Validate the header information is correct
+    // csvLine = header.getRow(1);
+    // EXPECT_PRED_FORMAT2(AssertQStringsEqual, csvLine[0], "caminfo");
+    // EXPECT_PRED_FORMAT2(AssertQStringsEqual, csvLine[2], "Viking1/VISB/33322515");
```

Comment:
Any reason to keep the old test commented out instead of removing it?

Reply:
Nope, just some stuff I was using for debugging that I missed